openpype/settings/ayon_settings.py

```diff
@@ -748,7 +748,20 @@ def _convert_nuke_project_settings(ayon_settings, output):
     )
 
     new_review_data_outputs = {}
-    for item in ayon_publish["ExtractReviewDataMov"]["outputs"]:
+    outputs_settings = None
+    # just in case that the users having old presets in outputs setting
+    deprecrated_review_settings = ayon_publish["ExtractReviewDataMov"]
+    current_review_settings = (
+        ayon_publish["ExtractReviewDataBakingStreams"]
+    )
+    if deprecrated_review_settings["outputs"] == (
```

Comment:
We only care about `current_review_settings["outputs"]` if they are the same as `deprecrated_review_settings["outputs"]`, so why not just do ` outputs_settings = deprecrated_review_settings["outputs"]` directly without the `if...else` ?

Reply:
i would do the same with `apply_setting` function in the extractor.